src/TwilioVideoLocalView.ios.js

```diff
@@ -19,6 +19,7 @@ class TwilioVideoLocalView extends Component {
   }
 
   render () {
+    scalesType = this.props.scaleType === 'fit' ? 1 : 2
```

Comment:
Not sure what this variable is for.

Reply:
Actually I'm a b it confused since on iOS you are expecting an int for `scalesType`.

Reply:
oddly the IOS setting requires an NSInteger, even though it looks like a string. It's an enum. I played around with sending the string and converting it in IOS, but couldn't quite figure it out, so I figured just convert it in react (which I know better) and send the int over the bridge.

Reply:
But how does scalesType actually make it into the video view?  should you be passing it in below, like

```
  <RCTTWLocalVideoView {...this.props} scalesType={scalesType}>
```

Reply:
Yes, on iOS it's an enum.  What issue were you running into when converting on iOS?  It should be pretty much the same as android.

Reply:
Yeah, you need to do a bit more bookkeeping to make it change dynamically, but that can wait for a future PR =).